### DISCUSSION TITLE
Changes in a working setup

### DISCUSSION THREAD

**Discussion by xploz1on** [evertramos/nginx-proxy-automation#353] [Q&A]
Hello everyone.
I came across a server that was already setup and have a working website with db. All i want to do is change the address is currently using. Right now its dev.example.com and its working with SSL and everything. I just want it to be example.com.
I could just manually edit the conf files but i believe this will mess up with the Lets Encrypt auto renew. So is there an option or easy way to make this changes? Thank you!

**Comment by evertramos** [comment]
Hello @xploz1on!

If you are using the latest version of nginx-proxy and nginx-proxy-automation, you might just change the url in config file and restart the service container (site container). 

Although your app/site must reply by this url in order to fully function your application. In WordPress for example, there is a few other settings you must change in database which I have it fully convered here:

https://github.com/evertramos/server-automation/blob/main/wordpress/update-site-domain.sh

But on the server side, changing the url in the environment variable of the container and restart will do the trick.

**Comment by xploz1on** [comment]
Hi thank for the reply. I added the new address tot he .env and in the wordpress site config.
So i still need to run the script? I was also wondering if i am able to add nginx proxy manager web ui?  Thanks!
